CHANGELOG.md

```diff
@@ -25,8 +25,10 @@ Piellorieppe is one of the main massif in the Sarek National Park.
 - [#175](https://github.com/nf-core/sarek/pull/175) - Add `Sentieon` documentation
 - [#176](https://github.com/nf-core/sarek/pull/176) - Add empty `custom` genome in `genomes.config` to allow genomes that are not in `AWS iGenomes`
 - [#179](https://github.com/nf-core/sarek/pull/179) - Add `FreeBayes` germline variant calling
-- [#180](https://github.com/nf-core/sarek/pull/180) - Now saving Mapped Bams (and creating TSV) in minimal setting
+- [#180](https://github.com/nf-core/sarek/pull/180) - Now saving Mapped BAMs (and creating TSV) in minimal setting
```

Comment:
Just out of curiousity, what is the minimal setting?

Reply:
Just a fasta file for the reference genome.
We have this user working on some Bees, so there is a reference genome, but nothing else, no know variants or such.
So the plan is actually to use sarek to do some variant calling to get some known variants (SNPs) from only duplicates marked files and then use these variants within sarek to be able to finally do some recalibration.

Reply:
ah, ok I see!

Reply:
cf https://gatkforums.broadinstitute.org/gatk/discussion/1247/what-should-i-use-as-known-variants-sites-for-running-tool-x

Reply:
yes, it looks like that...